include/llvm/Transforms/IPO/PassManagerBuilder.h

```diff
@@ -129,6 +129,8 @@ class PassManagerBuilder {
   bool HLSLHighLevel = false; // HLSL Change
   hlsl::HLSLExtensionsCodegenHelper *HLSLExtensionsCodeGen = nullptr; // HLSL Change
   bool HLSLResMayAlias = false; // HLSL Change
+  bool HLSLLiveValueAnalysis = false; // HLSL Change
+  std::string HLSLLiveValueAnalysisOutputFile = ""; // HLSL Change
```

Comment:
This requires an #include <string> above to make the Linux build happy. You might also consider using StringRef.

Reply:
Thanks, switched to StringRef.